src/components/structures/MessagePanel.tsx

```diff
@@ -252,17 +285,17 @@ export default class MessagePanel extends React.Component {
 
     /* return true if the content is fully scrolled down right now; else false.
      */
-    isAtBottom() {
-        return this._scrollPanel.current && this._scrollPanel.current.isAtBottom();
+    public isAtBottom(): boolean {
+        return this.scrollPanel.current?.isAtBottom();
     }
 
     /* get the current scroll state. See ScrollPanel.getScrollState for
      * details.
      *
      * returns null if we are not mounted.
      */
```

Comment:
if you feel bored, running over these legacy near-jsdoc comments with a manual formatter would be appreciated. Not critical though.